rasa_nlu/converters.py

```diff
@@ -160,12 +160,16 @@ def load_wit_data(filename):
     return TrainingData(training_examples)
 
 
-def load_markdown_data(filename):
+def load_markdown_data(filenames):
```

Comment:
Please also change the type annotation to `# type: (List[Text]) -> TrainingData`